ios-trivia-game/ViewControllers/AnswerViewController.swift

```diff
@@ -43,6 +45,13 @@ class AnswerViewController: UIViewController {
                 Logger.instance.log(logLevel: .info, message: "Adding \(answer.getJson())")
             }
             
+            // add the correct answer
+            let correctAnswer = self.question?.answer ?? ""
+            if (correctAnswer != "") {
+                let answer = Answer(answerText: correctAnswer)
```

Comment:
It's not necessary to store the correct answer to the database because every other user would send a request to store the correct answer in the database. The primary goal here is that the correct answer would be used on client-side, rather than being retrieved from the server-side for each user.

If the correct answer is stored in DB, then the scenario is we would have N number of correct answers, where N is the number of players because everyone would've submitted a correct answer from the question + their own answer. 

We can talk about this further.